v2/support/secure_settings_sample_app/README.md

```diff
@@ -0,0 +1,20 @@
+# Secure Settings Example App
+
+This sample app shows how to add Secure Settings to an app. "Secure Settings" is a neat feature that allows your App to pass a secure setting via the proxy that is invisible to normal agents.
```

Comment:
Could we include a link to the docs here? https://developer.zendesk.com/apps/docs/developer-guide/using_sdk#using-secure-settings

Reply:
Yes, I should do that! Done.